subprojects/smoke-test/build.gradle.kts

```diff
@@ -111,6 +106,8 @@ tasks {
     }
 
     val remoteProjects = withType<RemoteProject>()
+    val santaProjects = remoteProjects.matching { it.name.startsWith("santa") }
```

Comment:
I don't think we need to do String matching here. We can create a collection containing the two task providers, right?